Python/pywarpx/fields.py

```diff
@@ -77,8 +76,8 @@ class _MultiFABWrapper(object):
          Note that when True, the first n-ghost negative indices will refer to the lower
          ghost cells.
     """
-    def __init__(self, mf_name, level, include_ghosts=False):
-        self.mf_name = mf_name
+    def __init__(self, mf, level, include_ghosts=False):
+        self.mf = mf
         self.level = level
         self.include_ghosts = include_ghosts
 
```

Comment:
Maybe put the following lines in a method that can be called from here and from the inheritor (avoiding the code duplication).

Reply:
I changed things to actually call the `_MultiFABWrapper` constructor to also address the codeQL issue.